ui/apps/platform/src/Containers/Administration/Events/AdministrationEventHint.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { ReactElement } from 'react';
+import { List, ListItem, Text } from '@patternfly/react-core';
+
+/*
+ * Split hint string for conditional rendering in component:
+ * Return adjacent lines which start with hyphen space as array for bulleted list.
+ * Return other lines as string for br element if empty else paragraph.
+ */
+
+const listItemRegExp = /^- /;
+
+type LineOrList = string | string[];
+
+function splitHint(hint: string): LineOrList[] {
+    const lineOrListArray: LineOrList[] = [];
+    const listItemArray: string[] = [];
+
+    function pushListItems() {
+        if (listItemArray.length !== 0) {
+            lineOrListArray.push([...listItemArray]);
+            listItemArray.length = 0;
+        }
+    }
+
+    hint.split('\n').forEach((line) => {
+        if (listItemRegExp.test(line)) {
+            listItemArray.push(line.slice(2));
+        } else {
+            pushListItems();
+            lineOrListArray.push(line);
+        }
+    });
+    pushListItems();
+
+    return lineOrListArray;
+}
+
+export type AdministrationEventHintProps = {
+    hint: string;
+};
+
+function AdministrationEventHint({ hint }: AdministrationEventHintProps): ReactElement {
```

Comment:
This seems like a good candidate for 2 or 3 Jest snapshot tests.

Reply:
Super suggestion. Will do along with more integration tests next week.